parsons/databases/bigquery/bigquery.py

```diff
@@ -0,0 +1,968 @@
+import pickle
```

Comment:
blarg. I messed up the file renaming the first time.

here's a commit where I tried to (unsuccessfully) restore the file diff if it's helpful: https://github.com/move-coop/parsons/commit/fc2c9fc915eae9e70595a67da4e4ba3369ef8c7c